City-of-Pori/pori-kada
# PORI-286: Disable keyword links for now.

Disable visitpori keyword links until keyword listing theming has been done.

To test:
1. drush cc all
2. go to any attraction-card-listing and confirm that keywords are not clickable.
